<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	margin: 0;
	font-size: 10.1pt;
}
QComboBox{
 	padding: 2px 6px;
}
QPushButton{
  text-align: center;
  display: inline-block;
  padding: 2px 0;
  border: 1px solid #99a;
  border-bottom: 2px solid #99a;
  letter-spacing: 1px;
  overflow:hidden;
  margin: 0 10px;
  background-color: #fafaff;
  border-radius: 2px;
}
QPushButton:hover {
    background-color: #446;
    color: white;
	border: 1px solid #fff;
}
QPushButton::disabled {
    background-color: #ccc;
    color: #777;
	border: 1px solid #999;
}
</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kommentar (påkrevd hvis &amp;quot;Avvist&amp;quot;, valgfritt ellers):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Endre prosjektstatus&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Her kan du endre status til et prosjekt. Endringer du gjør her lagres først lokalt på PCen, og lastes opp når du trykker &amp;quot;Last opp...&amp;quot;.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="textComment"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="cmbSelectStatus"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Endre status til:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="btnSubmit">
     <property name="maximumSize">
      <size>
       <width>382</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Endre status</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelProject">
     <property name="text">
      <string>Hvilket prosjekt er valgt og hvilken status har det nå</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
